tests/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM google/cloud-sdk:latest
+FROM google/cloud-sdk:430.0.0
```

Comment:
It looks like latest is [431.0.0](https://hub.docker.com/r/google/cloud-sdk/tags), so we're actually downgrading.

Reply:
Right, which is the same as latest. I'll try to have Snyk ignore this one.